### DISCUSSION TITLE
How do launch Firefox with the tabs tree OPEN?

### DISCUSSION THREAD

**Discussion by DominikMendel** [mbnuqw/sidebery#1589] [General]
Title says it all? Whenever I start Firefox I would like to see the tab tree opened, even on newly created windows. How can I set this?

**Comment by mbnuqw** [comment]
Try to close all windows with sidebar open, then restart Firefox. Addons can't control this behavior, and Firefox _usually_ automatically opens the sidebar if previously closed window had it open.

**Reply by DominikMendel** [reply]
Ah this worked! Thanks!
